tests/AuthLog/AuthLogServiceTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * @copyright Copyright (c) Wizacha
+ * @license Proprietary
+ */
+declare(strict_types = 1);
+
+namespace Wizaplace\SDK\Tests\AuthLog;
+
+use Wizaplace\SDK\AuthLog\AuthLogService;
+use Wizaplace\SDK\Tests\ApiTestCase;
+
+class AuthLogServiceTest extends ApiTestCase
+{
+    public function testGet(): void
+    {
+        $authLogService = $this->buildAuthLogService();
+        $authLog = $authLogService->get(1);
+
+        $expected = [
+            'id' => 1,
+            'login' => 'hello@world.fr',
+            'status' => 'SUCCESS',
+            'source' => 'SSO',
+            'destination' => 'FRONT',
+        ];
+
+        $reality = [
+            'id' => $authLog['id'],
+            'login' => $authLog['login'],
+            'status' => $authLog['status'],
+            'source' => $authLog['source'],
+            'destination' => $authLog['destination'],
+        ];
+
+        $this->assertEquals($expected, $reality);
```

Comment:
```suggestion
        $this->assertEquals(
        [
            'id' => 1,
            'login' => 'hello@world.fr',
            'status' => 'SUCCESS',
            'source' => 'SSO',
            'destination' => 'FRONT',
        ],
        [
            'id' => $authLog['id'],
            'login' => $authLog['login'],
            'status' => $authLog['status'],
            'source' => $authLog['source'],
            'destination' => $authLog['destination'],
        ]);
```
Comme ça plus de variable intermédiaire :p